presto-druid/src/main/java/com/facebook/presto/druid/metadata/DruidSegmentInfo.java

```diff
@@ -42,7 +50,10 @@
 
     public enum DeepStorageType
     {
-        HDFS("hdfs");
+        HDFS("hdfs"),
+        S3_ZIP("s3_zip"),
```

Comment:
s/S3_ZIP/S3/g
zip is the file format, we just use S3 for storage type

Reply:
Updated enum variable name, while keeping the type name as defined from Druid.